rust/src/quic/frames.rs

```diff
@@ -346,8 +346,8 @@ fn parse_quic_handshake(msg: TlsMessage) -> Option<Frame> {
                     } else {
                         None
                     },
-                    ja4: if cfg!(feature = "ja4") {
-                        Some(ja4)
+                    hs: if cfg!(feature = "ja4") {
```

Comment:
Maybe we no longer need the `if cfg!(feature = "ja4")` here as we may want to log these parameters for quic...

Reply:
(maybe it should be in later `ja4 = JA4::try_new(rja4);` )